src/app/components/ThemeProvider/index.tsx

```diff
@@ -13,7 +14,8 @@ export const ThemeProvider = ({
   service,
   variant,
 }: PropsWithChildren<Props>) => {
-  let LoadableContextProvider;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let LoadableContextProvider: any = fallBackTheme;
```

Comment:
This took a long time to figure out.

Seems from `14.0.2` onwards the build step has changed in such a way that it expects some value in some cases. The ThemeProvider sits almost at the top level of the app, so it failed to build with errors like:
```
Error occurred prerendering page "/500". Read more: https://nextjs.org/docs/messages/prerender-error
ReferenceError: Cannot access 'v' before initialization
    at Object.Z (/Users/moorea53/Documents/GitHub/simorgh/ws-nextjs-app/build/server/chunks/themes-afaanoromoo.js:1:19139)
    at /Users/moorea53/Documents/GitHub/simorgh/ws-nextjs-app/build/server/chunks/themes-urdu.js:1:9592
    at r.a (/Users/moorea53/Documents/GitHub/simorgh/ws-nextjs-app/build/server/webpack-runtime.js:1:864)
    at 59186 (/Users/moorea53/Documents/GitHub/simorgh/ws-nextjs-app/build/server/chunks/themes-urdu.js:1:9236)
    at Function.r (/Users/moorea53/Documents/GitHub/simorgh/ws-nextjs-app/build/server/webpack-runtime.js:1:127)
```